CHANGES.rst

```diff
@@ -9,6 +9,12 @@ associations
 - Added new Lvl2 rule, Asn_Lv2NRSLAMPImage, to run Image2 pipeline for NRSLAMP
   exposures with OPMODE=image [#5740]
 
+combine1d
```

Comment:
minor nit, but the full name for the step has an underscore in it, i.e. "combine_1d" (like extract_1d)